pages/common/robo.md

```diff
@@ -0,0 +1,12 @@
+# robo
+
+> PHP task runner.
+> More information: <https://robo.li/>.
+
+- Run a specific command:
```

Comment:
Just want to clarify: Where does it find the tasks that it runs?

Reply:
Tasks are defined in a file called `RoboFile.php` in the current folder.
I am not sure there is a way to specify another path for this file.

Reply:
Ah, I see. Thanks for clarifying :-)